recipe_scrapers/kingarthur.py

```diff
@@ -22,7 +25,10 @@ def ingredients(self):
         return self.schema.ingredients()
 
     def instructions(self):
-        return self.schema.instructions()
+        schema_instructions = self.schema.instructions()
+        soup = BeautifulSoup(schema_instructions, "html.parser")
```

Comment:
Suggestion: could/should we add a comment here to explain that the site includes HTML content within the `schema.org` `recipeInstructions` element?

It seems strange that they're doing that, and I think there's a reasonable chance that they'll change that behaviour in future, so a comment could help explain to future maintainers why we did this.

Reply:
:+1: Added a comment to that affect, also added a fallback to the previous behavior, in case they do decide to change the behavior in the future, it won't completely fail.